Patch 1

HAJYOUNG opened the pull request:
Hello!
How have you been?
I changed some format of your website!
Can you pull it?!

HAJYOUNG commented:
Hello :)
Can you pull the files that I updated?
I'm doing the assignment about the Git, with your project.
I have to submit the assignment by today, so if you see this mail please push it.
Thank you and sorry..
Have a nice day!

meaghans commented:
> Hello!
> How have you been?
> I changed some format of your website!
> Can you pull it?!



> Hello :)
> Can you pull the files that I updated?
> I'm doing the assignment about the Git, with your project.
> I have to submit the assignment by today, so if you see this mail please push it.
> Thank you and sorry..
> Have a nice day!

Hello! Apologies for the late response. I checked out your updates and pulled the files, but it unfortunately doesn't mend well with how I prefer my website to look. The design is a little blocky and cluttered, which contrasts with the minimalist look that I was attempting to convey. I'm very sorry about this. If you are interested in continuing your collaboration, please let me know. I will gladly provide some more information about the design of my webpage!

HAJYOUNG commented:
I am sorry for lack of understanding of your website.
I'll mail you if there's a better idea next time.
Happy new year! Have a nice day:)
-----Original Message-----
보낸사람: "meaghans" <notifications@github.com>
받는사람: "meaghans/webpage" <webpage@noreply.github.com>
참조: "HAJYOUNG" <ga00881@naver.com>,"Author" <author@noreply.github.com>
날짜: 2019.11.26 오전 02:53:34 (GMT+09:00)
제목: Re: [meaghans/webpage] Patch 1 (#2)


Hello!
How have you been?
I changed some format of your website!
Can you pull it?!


Hello :)
Can you pull the files that I updated?
I'm doing the assignment about the Git, with your project.
I have to submit the assignment by today, so if you see this mail please push it.
Thank you and sorry..
Have a nice day!

Hello! Apologies for the late response. I checked out your updates and pulled the files, but it unfortunately doesn't mend well with how I prefer my website to look. The design is a little blocky and cluttered, which contrasts with the minimalist look that I was attempting to convey. I'm very sorry about this. If you are interested in continuing your collaboration, please let me know. I will gladly provide some more information about the design of my webpage!
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
